### DISCUSSION TITLE
Have we considered a `bentoml init` command?

### DISCUSSION THREAD

**Discussion by phitoduck** [bentoml/BentoML#3422] [Ideas]
Would it be valuable to add a `bentoml init` subcommand that creates a minimally working BentoML project?

React has `create-react-app`, AWS CDK has `cdk init --language=python`, `git` has `git init`. 

When starting from scratch, I find myself going into the docs or searching through past projects
to set up a basic

- `bentofile.yaml`
- `service.py` file with a runner and endpoint inside

For myself, I plan to write my own template with something like [Cookie Cutter](https://cookiecutter.readthedocs.io/en/stable/) or [`projen`](https://github.com/projen/projen). 

My team at work will definitely create a Bento template with some of our own opinions. Given this, I think there's a space the `bentoml` CLI tool could fill: a generic project that beginners can use to start learning, and experts can use to put up a project quickly.

Even one of the [bento examples](https://docs.bentoml.org/en/latest/tutorial.html#setup-for-the-tutorial) examples (like the [iris classifier](https://colab.research.google.com/github/bentoml/BentoML/blob/main/examples/quickstart/iris_classifier.ipynb#scrollTo=6458e417)) could be great.

**Comment by aarnphm** [comment]
It is in the backlog of our backlog. Would love to chat more on the design for your contributions :)
